docs/installation/installing.rst

```diff
@@ -111,15 +111,15 @@ Tip
     For more information, see :ref:`quickstart`.
 
 #.  Customize the KSQL ``<path-to-confluent>/etc/ksql/ksql-server.properties`` file.  The required parameters are ``bootstrap.servers``
-    and ``listeners``. You can also set any property the Kafka Streams API, the Kafka producer, or the Kafka consumer. 
+    and ``listeners``. You can also set any property the Kafka Streams API, the Kafka producer, or the Kafka consumer.
     For a description of common configurations and how to set them, see :ref:`configuring-ksql`.
 
     Here are the default settings:
 
     .. code:: bash
 
         bootstrap.servers=localhost:9092
-        listeners=http://localhost:8080
+        listeners=http://localhost:8088
```

Comment:
Question (this is unrelated/not touched by the PR): I noticed that the new `docs/installation/server-config/config-reference.rst` file documents `listeners` as `ksql.listeners`.  However, it appears to me as if all other parts of the docs + the code still use the shorter `listeners`.  Is this intentional?

Reply:
cc @joel-hamill

Reply:
@rodesai ^ wydt